src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -99,9 +100,18 @@ internal partial class CodeFixService : ICodeFixService
         public async Task<FirstFixResult> GetMostSevereFixAsync(
             TextDocument document, TextSpan range, ICodeActionRequestPriorityProvider priorityProvider, CodeActionOptionsProvider fallbackOptions, CancellationToken cancellationToken)
         {
-            var (allDiagnostics, upToDate) = await _diagnosticService.TryGetDiagnosticsForSpanAsync(
-                document, range, GetShouldIncludeDiagnosticPredicate(document, priorityProvider),
-                includeSuppressedDiagnostics: false, priorityProvider, DiagnosticKind.All, isExplicit: false, cancellationToken).ConfigureAwait(false);
+            var telemetryContext = "Pri" + (int)priorityProvider.Priority + "." + nameof(GetMostSevereFixAsync);
```

Comment:
I see that you added name of the method here in the telemetry context, but not at other places. I am fine with either approach of appending the containing method name or not, but we probably want to be consistent?

Reply:
Definitely agree that consistency is good, but from my viewpoint the most important point is that the telemetry data when viewed is informative and can be reasoned about. I tried to name the events as such in this PR, but I don't have the context which you have in this area so I'm definitely willing to change any of the event names if you think different names would be more informative. Not sure if you have looked at the dashboard with this information, as that might give a more holistic view of how the events relate.

Reply:
same comment about interpolated strings and interpolated string handlers.